vim_onenote.ahk

```diff
@@ -196,41 +295,70 @@ Send, {ShiftUp}
 return
 
 ; Delete current line
+; dd handled specially, to delete newline.
 d::
-if IsLastKey("d")
-{
-    Send, {Home}{ShiftDown}{End}
-    Send, ^c ; Yank before delete, don't use cut so blank lines are deleted 
+    SelectMotion()
+    send ^x
+    if IsLastKey("d")
     Send, {Del}
-    Send, {Shift}
-}
-return 
+return
 
-+s::
++S::
 Send, {Home}{ShiftDown}{End}
 Send, ^x ; Cut instead of yank and delete
 Send, {ShiftUp}
 Gosub, InsertMode   
 return 
 
-return 
 ; TODO handle regular paste , vs paste something picked up with yy
 ; current behavior assumes yanked with yy.
-p::
-Send, {End}{Enter}^v
-return
+p::Send, {End}{Enter}^v
+
 
-; Search 
+
+; Search actions
 /::
-Send, ^f
-Gosub InsertMode
+    Send ^f
+    GoSub, InsertMode
+    input, inp, E V, {escape}{return}
+    ; Send shift return to move back one search
```

Comment:
Updated comment.
the return key sent through to exit moves you forward once too many times.